Tools/CISupport/ews-build/results_db.py

```diff
@@ -96,18 +96,18 @@ def get_results_summary(cls, test, commit=None, configuration=None, logger=None)
                 response.deliverBody(cls.JsonPrinter(finished))
                 data = yield finished
                 defer.returnValue(json.loads(data))
-            else:
-                logger(f'Failed to query results summary with status code {response.code}\n')
-                defer.returnValue({})
+                return
```

Comment:
Can we simply `return` in this method or need to use `defer.returnValue({})`?

Reply:
We need to use `defer.returnValue`...that is the cost of the `defer.inline...` decorator, sadly